### PR TITLE
PD -2300,  PD-2195:  Refresh banner

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [PD-2300](https://jira.mongodb.org/browse/PD-2300)
🎟 _Jira ticket:_ [PD-2195](https://jira.mongodb.org/browse/PD-2195)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

**Review by shaneeza — COMMENTED** [review]
make sure all dependencies are up-to-date

File: packages/banner/src/Banner.tsx [hunk 1]
```diff
@@ -288,6 +48,18 @@ interface BannerProps extends HTMLElementProps<'div', never> {
...
+   *
+   * default: `false`
+   */
+  darkMode?: boolean;
+
+  /**
+   * The base font size of the title and text rendered in children.
+   */
+  baseFontSize?: BaseFontSize;
 }
```
Comment (spark33) [inline packages/banner/src/Banner.tsx:63]: This might belong in `types.ts`?  (Non-blocking)

File: packages/banner/src/Banner.tsx [hunk 2]
```diff
@@ -331,7 +104,13 @@ export default function Banner({
...
       className={cx(
         baseBannerStyles,
-        bannerVariantStyles[variant].body,
+        bodyTypeScaleStyles[baseFontSize],
+        bannerVariantStyles[theme][variant].body,
+        {
+          [css`
+            padding-right: 36px; // add space for the icon
+          `]: dismissible,
+        },
```
Comment (spark33) [inline packages/banner/src/Banner.tsx:113]: Could we break this out as something like `dismissibleBannerStyles` and use a `spacing` token?

**Review by spark33 — APPROVED** [review]
Minor comments, but non-blocking! Looks great.

File: packages/banner/src/Banner.tsx (hunk 2 above)
Comment (shaneeza) [reply packages/banner/src/Banner.tsx:113]: I can break this out but spacing doesn't have `36`

**Pull request merged**